### ISSUE TITLE
Patch 7.0.1 breaks packing 

### ISSUE THREAD

**Issue by rfoel** [juanjoDiaz/serverless-plugin-warmup#306]
For some reason when I update to 7.0.1 the serverless package command breaks.

Here's an exemple repo: https://github.com/rfoel/serverless-plugin-warmup-7.0.1-issue
Here's the working version with 7.0.0: https://github.com/rfoel/serverless-plugin-warmup-7.0.1-issue/tree/7.0.0

```
➜  aws-node-http-api-project git:(7.0.0) ✗ npx sls package --stage whatever

Packaging aws-node-http-api-project for stage whatever (us-east-1)
Warning: WarmUp: Skipping warmer "default" creation. No functions to warm up.
Environment: darwin, node 14.18.2, framework 3.1.1 (local), plugin 6.0.0, SDK 4.3.1
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
No file matches include / exclude patterns
```

**Comment by juanjoDiaz** [comment]
Good catch!

The issue only happens if the warmer there is no functions to warm up. In your case, because the warmer is not enabled.

I'll fix it in 7.0.2. But probably tomorrow.

**Comment by juanjoDiaz** [comment]
Closed with 7.0.2
